app/src/main/res/layout/activity_main.xml

```diff
@@ -5,15 +5,190 @@
     android:id="@+id/main"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/white"
     tools:context=".MainActivity">
 
-    <TextView
+    <View
+        android:id="@+id/top_view"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="@color/background"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHeight_percent="0.5"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_start"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="0.05" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline_end"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="0.95" />
+
+    <androidx.constraintlayout.helper.widget.Flow
+        android:id="@+id/flow_group"
+        android:layout_width="0dp"
+        android:layout_height="40dp"
+        app:layout_constraintTop_toTopOf="@id/top_view"
+        app:layout_constraintBottom_toBottomOf="@id/top_view"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintVertical_bias="0.067"
+        app:constraint_referenced_ids="text_settings,button_log_out" />
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/text_settings"
+        style="@style/TextAppearance.AppCompat.White24"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/settings"
+        app:layout_constraintTop_toTopOf="@id/flow_group"
+        app:layout_constraintStart_toStartOf="@id/flow_group"
+        app:layout_constraintBottom_toBottomOf="@id/flow_group"/>
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/button_log_out"
+        style="@style/ButtonStyle.LogOut"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/log_out"
+        app:cornerRadius="6dp"
+        app:strokeColor="@color/log_out"
+        app:strokeWidth="2dp"
+        app:layout_constraintTop_toTopOf="@id/flow_group"
+        app:layout_constraintEnd_toEndOf="@id/flow_group"
+        app:layout_constraintBottom_toBottomOf="@id/flow_group"/>
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/view_my_contacts"
```

Comment:
Rename this id. All widgets are inheretating from view, but this is button, hence id have to be "button" and not "view".

Reply:
done